Public/Src/Cache/ContentStore/Distributed/NuCache/RocksDbContentLocationDatabase.cs

```diff
@@ -36,17 +39,36 @@ public sealed class RocksDbContentLocationDatabase : ContentLocationDatabase
         private const string ActiveStoreSlotFileName = "activeSlot.txt";
         private StoreSlot _activeSlot = StoreSlot.Slot1;
         private string _storeLocation;
-        private readonly string _activeSlotFilePath;
+        private readonly string _activeSlotFilePath;        
+
+        private static readonly byte[] EmptyBytes = CollectionUtilities.EmptyArray<byte>();
 
         private enum StoreSlot
         {
             Slot1,
             Slot2
         }
 
+        /// <summary>
+        /// There's multiple column families in this usage of RocksDB.
+        ///
+        /// The default column family is used to store a <see cref="ContentHash"/> to <see cref="ContentLocationEntry"/> mapping, which has been
+        /// the usage since this started.
+        ///
+        /// All others are documented below.
+        /// </summary>
         private enum Columns
         {
-            ClusterState
+            ClusterState,
+            /// <summary>
+            /// Stores mapping from <see cref="StrongFingerprint"/> to a <see cref="ContentHashList"/>. This allows us
+            /// to look up via a <see cref="Fingerprint"/>, or a <see cref="StrongFingerprint"/>. The only reason we
+            /// can look up by <see cref="Fingerprint"/> is that it is stored as a prefix to the
+            /// <see cref="StrongFingerprint"/>.
+            ///
+            /// This serves all of CaChaaS' needs for storage, modulo garbage collection.
+            /// </summary>
+            CaChaaS

```

Comment:
Let's rename it to "Metadata" or "Fingerprints".

"Cachaas" is kind of an internal name that we use during discussion, but I don't think it is a good idea to use it in the code too much.